rules/legacy_xcodeproj.bzl

```diff
@@ -1085,7 +1085,7 @@ def _xcodeproj_impl(ctx):
     ctx.actions.run(
         executable = ctx.executable._xcodegen,
         arguments = ["--quiet", "--no-env", "--spec", xcodegen_jsonfile.path, "--project", project.dirname],
-        inputs = [xcodegen_jsonfile],
+        inputs = [xcodegen_jsonfile] + ctx.files.additional_files,
```

Comment:
Mind clarifying why do you need this? Isn't the json now taking the paths from the updated `project_file_groups`? Meaning that changes in that file group changes the JSON and already invalidate this action?

Reply:
Without this change, the standalone build is fine, but the sandbox build will break complaining `Spec validation error: Invalid file group "../../../../../../tests/ios/frameworks/objc/additional_files/foo.podspec"`. 

I have to add this change to pass the tests in rules-ios, but i am also not sure if this is the correct fix. Do you have any suggestion where should I declare these files correctly?